packages/oidf-resolution-tests/__tests__/restAgent.test.ts

```diff
@@ -0,0 +1,118 @@
+import { IdentifierResolution, IIdentifierResolution } from '../../identifier-resolution/src' // FIXME fix when new types have been absorbed throughout ssi-sdk
+import { SphereonKeyManager } from '@sphereon/ssi-sdk-ext.key-manager'
+import { SphereonKeyManagementSystem } from '@sphereon/ssi-sdk-ext.kms-local'
+
+import { createAgent, IAgent, IAgentOptions, IKeyManager, TAgent } from '@veramo/core'
+import { Entities, KeyStore, migrations, PrivateKeyStore } from '@veramo/data-store'
+import { SecretBox } from '@veramo/kms-local'
+import { AgentRestClient } from '@veramo/remote-client'
+import { AgentRouter, RequestWithAgentRouter } from '@veramo/remote-server'
+import { OrPromise } from '@veramo/utils'
+
+// @ts-ignore
+import express from 'express'
+import { Server } from 'http'
+import { DataSource } from 'typeorm'
+
+import oidfResolutionTests from './shared/oidfResolutionTest'
+import { ResourceResolver } from '@sphereon/ssi-sdk.resource-resolver'
+import { OIDFClient } from '@sphereon/ssi-sdk.oidf-client'
+import { IJwtService, JwtService } from '@sphereon/ssi-sdk-ext.jwt-service'
+
+jest.setTimeout(30000)
+
+const databaseFile = ':memory:'
+const port = 13213
+const basePath = '/agent'
+
+119f7e793cc2899e99b64b88ca8603db986700089532b'
```

Comment:
Again check your code and the build output

Reply:
Fixed already.
Bram requested to remove these, somehow IJ must have messed up my selection